contracts/exchange/test/reentrancy_tests.ts

```diff
@@ -0,0 +1,120 @@
+import { blockchainTests, constants, describe, expect, hexRandom } from '@0x/contracts-test-utils';
+import { BigNumber } from '@0x/utils';
+import { DataItem, MethodAbi, TupleDataItem } from 'ethereum-types';
+import * as _ from 'lodash';
+
+import { artifacts, ReentrancyTesterContract } from '../src';
+
+import { constants as TestConstants } from './utils/constants';
+
+blockchainTests.resets('Reentrancy Tests', env => {
```

Comment:
Nice! Having this all in one place is so much cleaner.

Reply:
➕